os/drivers/lwnl/lwnl.c

```diff
@@ -144,20 +144,39 @@ static int lwnl_open(struct file *filep)
 	return ret;
 }
 
-
 static int lwnl_close(struct file *filep)
 {
 	LWNL_ENTER;
 	struct inode *inode = filep->f_inode;
 	struct lwnl_upperhalf_s *upper = inode->i_private;
 	int ret = OK;
+#ifndef CONFIG_DISABLE_POLL
+	FAR struct lwnl_open_s *opriv = &upper->ln_open;
```

Comment:
it's not private data of upper, renaming should be done.

Reply:
I don't understand your comment. Could you explain more? It is used internally.

Reply:
It's not major comment, but priv being used for private context data, usually.
But ln_open is defined explicitly, it's not good naming.
For example, priv is dev_a's priv pointer in below..
```
 struct dev_a {
  int type;
  int value;
  void *priv;
}

struct lowhalf_a {
   int size;
   int length;
}

struct dev_a *g_dev;

struct a initalize_driverA(lowhalf_a *dev) {
  g_dev->priv = dev;
  register_driver(path, &dev_aops, 0666, g_dev);
  
}

// dev_aops 
static int dev_a_open(FAR struct file *filep)
{
	FAR struct inode *inode = filep->f_inode;
	FAR struct dev_a *dev = inode->i_private;
	FAR struct lowhalf_a *priv = dev->priv;
	...
}
``` 
```

Reply:
To sum up, priv is being used for iNode's private data(upper) or private context data(lower) of private data(upper)

Reply:
Thank you for explanation. I think it is just a data of upper struct, not private context data(lower) as you said. I will rename it.